processor/src/tasks/processGerbers/index.ts

```diff
@@ -231,51 +231,47 @@ export default async function processGerbers(
 }
 
 async function generateTopLargePng(topSvgPath, stackup, topLargePngPath) {
-  let cmd_large = `inkscape --without-gui '${topSvgPath}'`
-  cmd_large += ` --export-png='${topLargePngPath}'`
+  let exportWidth: number
   if (stackup.top.width > stackup.top.height + 0.05) {
-    cmd_large += ` --export-width=${240 * 3 - 128}`
+    exportWidth = 240 * 3 - 128
   } else {
-    cmd_large += ` --export-height=${180 * 3 - 128}`
+    exportWidth = 180 * 3 - 128
   }
-  await exec(cmd_large)
+  await sh`inkscape --without-gui ${topSvgPath} --export-width=${exportWidth} --export-png=${topLargePngPath}`
   await s3.uploadFile(topLargePngPath, 'image/png')
```

Comment:
Is converting both branches to `--export-width` intentional?

Reply:
No, nice catch!

Reply:
Fixed with 068f4ca826648ca32